containers/podman-desktop
# fix: update cli management in kind extension

### What does this PR do?

It updates the kind extension to use the new install/update/uninstall features for cli tools

### Screenshot / video of UI

![kind_cli_tool](https://github.com/user-attachments/assets/74f3928a-8767-4bf2-8e0a-b3c4d6194032)

### What issues does this PR fix or reference?

it resolves #8609 

### How to test this PR?

1. try to install/upgrade/uninstall kind
2. if kind has been installed by the user in a different folder, you should not be able to update/uninstall it
